fmriprep/utils/misc.py

```diff
@@ -7,6 +7,28 @@
 """
 
 
+def remove_rotation_and_shear(img):
+    from transforms3d.affines import decompose, compose
+    import numpy as np
+
+    T, _, Z, _ = decompose(img.affine)
+    affine = compose(T=T, R=np.diag([1, 1, 1]), Z=Z)
+    return img.__class__(np.asanyarray(img.dataobj), affine, img.header)
+
+
+def split_and_rm_rotshear_func(in_file):
+    import os
+    from nilearn.image import iter_img
```

Comment:
Is there any memory-related reason to use nilearn's `iter_img` which has quite a bit of overhead, instead of [`nibabel.func.four_to_three`](http://nipy.org/nibabel/reference/nibabel.funcs.html#four-to-three)?
```suggestion
    import nibabel as nb
```